saleor/graphql/webhook/subscription_payload.py

```diff
@@ -98,7 +98,6 @@ def _get_plugins(requestor_getter):
     request.requestor = requestor  # type: ignore
     request.request_time = request_time  # type: ignore
     request.site = SimpleLazyObject(lambda: Site.objects.get_current())  # type: ignore
-    request.plugins = SimpleLazyObject(lambda: _get_plugins(requestor))  # type: ignore
```

Comment:
What about this? What is a reason for removing this? Did you check this with webhook that has subscription query defined?

Reply:
Well, all tests pass (not counting the ones that tested middleware). The whole point of my work is to get rid of each LazyObject attached to request/context. Each mention of `request.plugins` or `context.plugins` has been replaced with `load_plugins(...)` call.

Reply:
Are we sure the tests are actually covering this? I would double-check manually as Maciej suggests.

Reply:
This is a good case to push for 100% coverage and mutation tests :wink: